projects/plugins/jetpack/modules/search/instant-search/components/dom-event-handler.jsx

```diff
@@ -183,11 +183,13 @@ export default class DomEventHandler extends Component {
 	};
 
 	scrollOverlayToTop = event => {
-		// NOTE: the propertyName need to be aligned with the animation
-		if ( event?.propertyName !== 'opacity' ) {
+		// NOTE: the propertyName need to be aligned with the animation: instant-search/components/overlay.scss.
+		// We don't scroll the overlay on overlay close.
+		if ( event.propertyName !== 'opacity' || ! this.props.isVisible ) {
```

Comment:
And so sorry to make you feel bad about this line - it was right in the previous PR, because we want to return early if it's not `opacity`... :neckbeard: